cypress/Objects/TimeLoggin.js

```diff
@@ -0,0 +1,25 @@
+class TimeLoggin{
+    urlIncludeTimeLogging(){
+        return cy.url()
```

Comment:
from the name of your function I would assume you actually perform some kind of validation that it includes time logging url.

Overall function that calls another function without any parameters is kind of useless, you could just call cy.url() instead.

what I would suggest to do is this approach:
validateUrl() {
      cy.url().should('include', '/time-logging')
}